inspirehep/modules/workflows/tasks/matching.py

```diff
@@ -127,9 +128,9 @@ def fuzzy_match(obj, eng):
 
     fuzzy_match_config = current_app.config['FUZZY_MATCH']
     matches = dedupe_list(match(obj.data, fuzzy_match_config))
-    record_ids = [el['_source']['control_number'] for el in matches]
-    obj.extra_data.setdefault('matches', {})['fuzzy'] = record_ids[0:5]
-    return bool(record_ids)
+    record_refs = [get_record_ref(el['_source']['control_number'], 'literature') for el in matches]
```

Comment:
hmm, the default value, ``record``, should be enough https://github.com/inspirehep/inspire-dojson/blob/master/inspire_dojson/utils/__init__.py#L134

Reply:
It isn't: look where https://labs.inspirehep.net/api/record/4328 redirects to.

Reply:
Is it any special requirement for that? It seems to me very strange to redirect the api to a html page.